src/main/java/com/conveyal/r5/streets/LinkedPointSet.java

```diff
@@ -56,10 +56,12 @@
      */
     public int[] edges;
 
-    /** For each point, distance from the initial vertex of the edge to the split point. */
+    /** For each point, distance from the beginning vertex of the edge geometry up to the split point for a link point,
+     * plus the distance from the link point to the split point */
```

Comment:
I'm not sure the different meanings of "link point" and "split point" will be clear to a future reader.